samples/bot-conversation/python/teamsapp.yml

```diff
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: 1.0.0
+
```

Comment:
Also add `sampleTag` to the yml file, refer the nodejs sample for example

Reply:
done.